scheduler/ecs/ecs.go

```diff
@@ -47,6 +47,7 @@ type Scheduler struct {
 
 	cluster string
 	ecs     *ecsutil.Client
+	config  Config
```

Comment:
I don't believe the `Scheduler` struct every needs to directly access the log driver, just the ProcessManager. This can be removed.